app/fonts.styl

```diff
@@ -0,0 +1,64 @@
+@font-face {
```

Comment:
That is because some of the styles were copied 'as is' from scss files. That doesn't mean that they won't be rewritten in `stylus` in future. They will be rewritten when we'll have implemented all the pages given. Treat it as a `hack` for now :)